Add `osu! Language Exchange: September 2023` news post (#9941)
ppy/osu-wiki: news/2023/2023-08-20-osu-language-exchange-september-2023.md

## Achievements

Throughout the event, there will be achievements for you to complete that require you to do different activities with your language exchange partners. Some examples include having a certain amount of conversations, writing a certain amount of short stories, or even doing cooking streams in call with your language partners. The full list of achievements can be found on the event's [forum post](https://osu.ppy.sh/community/forums/topics/1808561).

These achievements are designed to give you some activities to do, but your achievements will not be checked by the event staff and there will be no prizes for completing them. Have fun!

## Progress logs

On each day of the month, you will be able to submit a very simple progress log to help you keep track of your progress. The progress log will ask you what you focused on that day and how you felt about that day overall.

If you submit at least 10 progress logs, you will be given a special *Linguist* role in the Discord server. This is simply designed to give a small incentive and help you stay motivated in your studies! Completing progress logs is not required to participate in the event.

---

> Interested in joining, but feel like you might not have enough time? No worries! The [sign-up form](https://forms.gle/GiZUvqu4XQtjidMt5) will ask you for your availability so you can take the event at your own pace.

Reviewer: ```suggestion
Interested in joining, but feel like you might not have enough time? No worries! The [sign-up form](https://forms.gle/GiZUvqu4XQtjidMt5) will ask you for your availability, so you can take the event at your own pace.
```

If you'd like to share this event with someone in a different language, you can find translated versions of this news post at the bottom of the English forum post [here](https://osu.ppy.sh/community/forums/topics/1808561).

If you're ready to sign up, you can fill out the sign-up form in English [here](https://forms.gle/GiZUvqu4XQtjidMt5) or in another language through any one of the language-specific forum posts.

---

> If you know anyone who is interested in improving their skills in a language (or learning a new one), please consider sharing this event with them! This event will work best with as many sign-ups as possible, so we hope to see you all join us next month for the first edition of OLEX.

Reviewer: LTeX is flagging as too many repetitions of "if" here, might need a reword on either of the 3 sentences?

Reviewer: LTeX is flagging as too many repetitions of "if" here, might need a reword on either of the 3 sentences?

```suggestion
In case that you know anyone who is interested in improving their skills in a language (or learning a new one), please consider sharing this event with them! This event will work best with as many sign-ups as possible, so we hope to see you all join us next month for the first edition of OLEX.
```

Author: looks good but the "that" can be removed

Author: ```suggestion
in case you know anyone who is interested in improving their skills in a language (or learning a new one), please consider sharing this event with them! This event will work best with as many sign-ups as possible, so we hope to see you all join us next month for the first edition of OLEX.
```
